test/unit/hello.world.controller.test.ts

```diff
@@ -0,0 +1,72 @@
+import { HelloWorldController } from "../../src/controllers/hello-world";
+import { UserProfile } from "@loopback/authentication";
+import { expect, sinon } from "@loopback/testlab";
+import {
+  DefaultCrudRepository,
+  DataSourceConstructor
+} from "@loopback/repository";
+import { UserRepository } from "../../src/repositories/user.repository";
+
+describe("HelloWorldController", () => {
```

Comment:
I think a better file name would be `test/unit/controllers/hello-world.controller.test.ts`. Since each unit test should be testing only a single unit (class, source code file), a directory structure copying `src` structure make more sense to me.